tests/integration/testdata/dump/009-skip-defaults/enterprise/3.10+/plugin-partial-2.expected.yaml

```diff
@@ -13,18 +13,6 @@ plugins:
     - abc-client
     client_secret:
     - abc-secret
-    cluster_cache_redis:
-      timeout: 2000
     issuer: http://example-issuer.com
-    redis:
-      timeout: 2000
-    session_redis_cluster_max_redirections: 5
```

Comment:
These were shorthand_fields. Hence, with the current change, they would no longer show up.